test/metabase/api/user_test.clj

```diff
@@ -495,8 +496,9 @@
 ;;; |                                      Updating a User -- PUT /api/user/:id                                      |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
-(defn include-personal-collection-name
-  {:hydrate :personal_collection_name}
+(mi/define-simple-hydration-method include-personal-collection-name
+  ::personal-collection-name
```

Comment:
I changed the hydration key here to a namespaced keyword so it was clear that it was only used in this test namespace and we weren't accidentally testing that it came back with API responses or anything like that